tests/testthat/test-tamper.R

```diff
@@ -0,0 +1,82 @@
+
+context("Tampering")
+
+test_that("We can detect if we are in a pipe", {
+
+  skip(message = "not ready yet for a real test")
```

Comment:
Yes, that's on the TODO list, to have the parallel of `utils::dump.frames()` that uses the same machinery as `tamper()`. Maybe called `dump_pipes()`. To test the interactive behavior seems *very* hard, so I say it is not worth the effort. :)